scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/KafkaAdmin.kt

```diff
@@ -105,5 +105,6 @@ class KafkaAdmin(
 
     companion object {
         private val logger = coLogger(KafkaAdmin::class)
+        private val noCoLogger = noCoLogger(KafkaAdmin::class)
```

Comment:
we need this because in the new version of kotlin-retry, the retry policy is no longer executed in coroutine context.